packages/dds/tree/src/test/simple-tree/utils.ts

```diff
@@ -145,6 +147,16 @@ export function hydrate<TSchema extends ImplicitFieldSchema>(
 	return getTreeNodeForField(field) as TreeFieldFromImplicitField<TSchema>;
 }
 
+/**
+ * {@link hydrate} but unsafe initialTree
```

Comment:
When should this be used? Are the test usages introduced in this PR valid? Or are they indicative of cleanup work that needs to be done?

Reply:
Details added.